examples/CartPole/DQN.jl

```diff
@@ -48,14 +48,11 @@ remember(state, action, reward, next_state, done) =
   push!(memory, (data(state), action, reward, data(next_state), done))
 
 function action(state, train=true)
-  train && rand() <= get_ϵ(e) && (return rand(-1:1))
+  train && rand() <= get_ϵ(e) && (return Gym.sample(env.action_space))
```

Comment:
env.action_space -> env._env.action_space

Reply:
I'm not so sure about that. The action space is already a part of the `EnvWrapper`. And there's nothing that you can do with the action space, except for using them to draw samples and check whether the entered action is valid or not. Adding another helper function  that just returns the action space of the environment seems unnecessary.

Reply:
Could not find `action_space` as part of `EnvWrapper`. Where did you find it?

Reply:
https://github.com/kraftpunk97/Gym.jl/commit/b9b950e6fa2784d7526499b6e5279c01f6cbac39